models/aeif_cond_beta_multisynapse.h

```diff
@@ -51,20 +51,66 @@
 
  Description:
 
- aeif_cond_beta_multisynapse is an extension of
- aeif_cond_alpha_multisynapse.  It allows an arbitrary number of synaptic
- rise time and decay time constant. Synaptic conductance is modeled by a
+ aeif_cond_beta_multisynapse is a conductance based adaptive exponential
+ integrate-and-fire neuron model. It allows an arbitrary number of synaptic
+ rise time and decay time constants. Synaptic conductance is modeled by a
  beta function, as described by A. Roth and M.C.W. van Rossum
  in Computational Modeling Methods for Neuroscientists, MIT Press 2013,
  Chapter 6
 
- The time constants are supplied by two arrays taus_rise and taus_decay for
- the synaptic rise time and decay time, respectively.  Port numbers
- are then automatically assigned and there range is from 1 to n.  (n
- being the index of the last element of the tau_ex and tau_in
- arrays).
+ The number of receptor ports is supplied by the function "n_receptors".
```

Comment:
I would drop this sentence, and `n_receptors` is not a function. I think the meaning of n_receptors becomes clear from the last sentence in this paragraph.